client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx

```diff
@@ -28,11 +32,10 @@ function PostActionsEllipsisMenuDuplicate( {
 	type,
 	duplicateUrl,
 	onDuplicateClick,
-	calypsoifyGutenberg,
 } ) {
 	const validStatus = includes( [ 'draft', 'future', 'pending', 'private', 'publish' ], status );
 
-	if ( ! canEdit || ! validStatus || 'post' !== type || calypsoifyGutenberg ) {
+	if ( ! canEdit || ! validStatus || 'post' !== type ) {
```

Comment:
We should check here if the Copy post module is active and don't allow to copy/duplicate if it is deactivated. Otherwise the new posts won't contain content and this will be the default behavior since the Copy post module is not activated by default.

Reply:
Good idea; it also means we can just have one check (for the activated module), and not require the version check any more.

So bear with me – I've used `isUnsupportedJetpack` (instead of `isSupportedJetpack`), since `! isSupportedJetpack` would be truthy for both Jetpack sites that aren't supported, and simple sites. Let me know if there's a more clear logical way to express this 🙃